Modules/Core/Common/test/itkGaussianDerivativeOperatorTest.cxx

```diff
@@ -159,5 +177,13 @@ itkGaussianDerivativeOperatorTest(int argc, char * argv[])
     std::cout << "STREAMED ENUM VALUE GaussianDerivativeOperatorEnums::InterpolationMode: " << ee << std::endl;
   }
 
-  return EXIT_FAILURE;
+  std::cout << "Test finished." << std::endl;
+  if (testStatus)
+  {
+    return EXIT_SUCCESS;
+  }
+  else
+  {
+    return EXIT_FAILURE;
+  }
```

Comment:
`return testStatus ? EXIT_SUCCESS : EXIT_FAILURE;`

Reply:
The ITK SW Guide discourages the use of the ternary operator.